css/style.css

```diff
@@ -0,0 +1,66 @@
+body {
+    background-color: #39393A;
+    color: #C5DECD;
+    font-family: 'Open Sans', sans-serif;
+    font-size: 28px;
+}
+
+h1, h2 {
+    font-family: 'Jost', sans-serif;
+    font-size: 38px;
+}
+
+p {
+    color: #FDCA40;
+}
+
+li {
+    list-style-type: square;
+    color: #AEDD92;
+}
+
+img {
+    margin-left: 5px;
+}
+
+input {
+    padding: 10px;
+    border-radius: 5px;
+}
+
+input[type=text] {
+    width: 250px;
+    border: none;
+}
+
+input[type=submit], button {
+    color: #39393A;
+    border: none;
+    font-size: 18px;
+}
+
+input[type=submit] {
+    background-color: #C5DECD;
+}
+
+button {
+    background-color: #FDCA40;
+}
+
+input[type=submit]:hover, button:hover{
+    background-color: #FFFFFF;
```

Comment:
If you're interested, a shorthand for hex values in CSS where all 6 characters is the same is to just use three characters - `#FFF` is shorthand for `#FFFFFF`.